src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
> why that name mangling wasn't happening when these functions were in the existing .cpp file

Make sure to include the header file that declares the method as extern "C" - it is what was happening before.